src/routes/api/index.js

```diff
@@ -1,13 +1,13 @@
 import express from 'express';
 import authRouter from './auth';
 import requestRouter from './request';
+import commentRouter from './comment';
 import { checkToken } from '../../middlewares';
 
 const router = express.Router();
 
 router.use('/auth', authRouter);
-
-router.use('/request', checkToken, requestRouter);
+router.use('/request', checkToken, requestRouter, commentRouter);
```

Comment:
Great work bro. Nicely done. It was really easy to read through and understand what you've done. The only thing that I feel might cause problems is where you placed the comment Router. I feel following the REST API convention the route should be a POST request at /api/v1/comment. This clearly conveys that it is creating a new comment. Having both a request and comment router on one endpoint doesn't look clean.

If its separated, it would make it easier in the future to add PATCH, DELETE functionalities to the comment route

Reply:
How does this even work? Things are happening?

Reply:
And @EziamakaNV , it is surely a post request
the route is POST /api/v1/requests/:requestId/comments, this is a convention too because the comment(s) belong to a User and a Request